CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Added
+- New endpoint for Convert:
+   - GET /sapi/v1/convert/tradeFlow to support user query convert trade history records
```

Comment:
sorry, forgot to mention about the right indention and use of quote marks here, please take example from previous additions :)